src/schemas/swap_quote_request_schema.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "/SwapQuoteRequestSchema",
+    "properties": {
+        "sellToken": {
+            "type": "string"
+        },
+        "buyToken": {
+            "type": "string"
+        },
+        "takerAddress": {
+            "$ref": "/addressSchema"
```

Comment:
does the `addSchema` thing allow us to reference `/addressSchema`'s relative path from `json-schemas`?

Reply:
Yes exactly, unused now, probably needed in the future. There was an open question on "how this is done".